docs/specification/json-schemas.md

```diff
@@ -0,0 +1,31 @@
+# Json-schema definitions
+
+These schemas are offered for those who want to use ValueFlows but are not using Linked Open Data / Semantic Web technology.  See https://json-schema.org/ for more information.
```

Comment:
I think we need to include here something like:
**VF doesn't require any specific serialization, publishers can publish valid VF data using any of [RDF1.1 Serializations](https://www.w3.org/TR/rdf11-new/#section-serializations) including [JSON-LD](https://json-ld.org/). Even valid VF data published in JSON-LD format doesn't need to follow those schemas. Applications assuming use of those schemas will only interoperate with the subset of VF ecosystem which uses JSON and conforms to schemas below.**

Reply:
Sure I'll work something like that in somewhere, maybe a more general page?

Reply:
I think we need both. A more general page elaborating in depth on RDF, JSON-LD, JSON, JSON Schemas, Media Types etc.
Each specification page might have a shorter note like the one above which links to that more general in-depth page.

Reply:
:+1:

Reply:
I'll try to find a little bit of time today to make separate PR with dedicated page. This PR would just need to include a paragraph in lines of what I proposed above and link to that dedicated page.